tests/e2e/deployment/rule_crd_test.go

```diff
@@ -0,0 +1,146 @@
+package deployment
+
+import (
+	"bytes"
+	"net/http"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	v1 "github.com/kubeedge/kubeedge/cloud/pkg/apis/rules/v1"
+	"github.com/kubeedge/kubeedge/pkg/apis/componentconfig/cloudcore/v1alpha1"
+	"github.com/kubeedge/kubeedge/tests/e2e/utils"
+)
+
+const (
+	RuleEndpointHandler = "/apis/rules.kubeedge.io/v1/namespaces/default/ruleendpoints"
+	RuleHandler         = "/apis/rules.kubeedge.io/v1/namespaces/default/rules"
+)
+
+var _ = Describe("Rule Management test in E2E scenario", func() {
```

Comment:
Do the e2e tests have been run？We should start them.

Reply:
done